nikola/data/themes/base/assets/css/rst.css

```diff
@@ -47,51 +63,23 @@ div.abstract p.topic-title {
   text-align: center }
 
 div.admonition, div.attention, div.caution, div.danger, div.error,
-div.hint, div.important, div.note, div.tip, div.warning, div.sidebar {
-/* stolen from Boostrap 3 (.panel .panel-default) */
```

Comment:
I’d like those bootstrap-esque styles to appear in bootstrap3 (eg. `theme.css` or in an override of `rst.css`)